pdc-contact-browser.spec

```diff
@@ -22,6 +22,41 @@ cp -R var/ %{buildroot}/
 /var/www/html/%{name}
 
 %changelog
+* Wed Jan 24 2018 Lubomír Sedlář <lsedlar@redhat.com> 2.0.0-1
+- Bump version (lsedlar@redhat.com)
```

Comment:
It would be better to clean up the changelog a bit. Not every commit is relevant to changes from last version.